packages/components/OmniLink.tsx

```diff
@@ -0,0 +1,61 @@
+import { useLinkProps } from "@react-navigation/native";
+import React, { ReactNode } from "react";
+import {
+  Platform,
+  StyleProp,
+  TouchableOpacity,
+  TouchableOpacityProps,
+  View,
+  ViewStyle,
+} from "react-native";
+
+export interface OmniLinkToType {
+  screen: string;
+  params?: object;
+}
+
+export const OmniLink: React.FC<{
+  to: OmniLinkToType;
+  action?: any | undefined;
+  children: ReactNode | undefined;
+  style?: StyleProp<ViewStyle | TouchableOpacityProps>;
+
+  disabled?: boolean;
+}> = ({ to, action, children, style, disabled, ...rest }) => {
+  // @ts-expect-error
+  const { onPress, ...props } = useLinkProps({ to, action });
+
+  const [isHovered, setIsHovered] = React.useState(false);
+
+  if (Platform.OS === "web") {
+    // It's important to use a `View` or `Text` on web instead of `TouchableX`
+    // Otherwise React Native for Web omits the `onClick` prop that's passed
+    // You'll also need to pass `onPress` as `onClick` to the `View`
+    // You can add hover effects using `onMouseEnter` and `onMouseLeave`
+    return (
+      <View
+        // @ts-expect-error
```

Comment:
should not hide error

Reply:
improved with a comment, but this is `required`

Reply:
@sakul-budhathoki , @hthieu1110 recommended me your expertise.

What do you think about this line?

I believe that is a limitation of the ts linter, as it does not understand this specific case.
I don't think that is taking into account the `if` I have in place.

If I truly want to fix this, I would just change it to be a native web (`<a>`) element and not a react native element `<View>`